DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Administrator/SearchableAdminViewModel.cs

```diff
@@ -6,14 +6,15 @@
 
     public class SearchableAdminViewModel : BaseFilterableViewModel
     {
-        public SearchableAdminViewModel(AdminUser adminUser)
+        public SearchableAdminViewModel(AdminUser adminUser, bool hasSuperAdminAccess, int currentAdminUserId)
         {
             Id = adminUser.Id;
             Name = adminUser.SearchableName;
             CategoryName = adminUser.CategoryName ?? "All";
             EmailAddress = adminUser.EmailAddress;
             IsLocked = adminUser.IsLocked;
-            IsCentreManager = adminUser.IsCentreManager;
+            CanShowDeactivateAdminButton = hasSuperAdminAccess
+                                           && currentAdminUserId != adminUser.Id;

```

Comment:
Am I right in thinking that this will prevent _any_ user apart from super-admins from deactivating this admin?

Reply:
Yes, that's correct.

Reply:
That's not quite what we want.
The ticket says, 
* "This page should be accessible only by the centre manager or a super admin."
* "Also, remove the Deactivate Admin Account button when the user [on the card] is a centre manager (requested by HEE in show and tell 30/9/2021)."
* "Also, remove the Deactivate Admin Account button when the user [on the card] is a super-admin (unless the logged-in-user a different super admin) (requested by HEE in daily catchup 25/11/2021)."

Reply:
These requirements have been clarified on the ticket: https://softwiretech.atlassian.net/browse/HEEDLS-594